.github/workflows/run-unit-tests.yml

```diff
@@ -33,3 +33,26 @@ jobs:
     - name: Test with pytest
       run: |
         pytest
+        
+  build-python-legacy:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6]
+        os: [ubuntu-20.04, macOS-latest, windows-latest ]
+
+    steps:
```

Comment:
Can you provide clarity on why the steps changed when they worked before. Can we not keep the tests the same with the old ubuntu?

Reply:
Based on the ongoing issue (see description). Python 3.6 is no longer supported in "ubunto-latest". Because of this, in order to successfully run 3.6 in ubunto, we must explicitly refer to the last known working version of ubunto (in this case 20.04)

Reply:
To clarify, the actual tests have not changed. This change is a change to the GH actions